lldb/include/lldb/Symbol/SymbolFile.h

```diff
@@ -475,6 +484,8 @@ class SymbolFile : public PluginInterface {
 private:
   SymbolFile(const SymbolFile &) = delete;
   const SymbolFile &operator=(const SymbolFile &) = delete;
+
+  static LocateDwoCallback LOCATE_DWO_CALLBACK;
```

Comment:
static variable are named starting with "s_" and then use lowercase separated by _. So this should be:
```
static LocateDwoCallback s_locate_dwo_callback;
```